webknossos/webknossos/dataset/dataset.py

```diff
@@ -341,17 +349,17 @@ def __init__(
             self._layers[layer_properties.name] = layer
 
         if dataset_existed_already:
-            if voxel_size is None:
+            if voxel_size_with_unit is None:
                 warnings.warn(
-                    "[DEPRECATION] Please always supply the voxel_size when using the constructor Dataset(your_path, voxel_size=your_voxel_size)."
+                    "[DEPRECATION] Please always supply the voxel_size or voxel_size_with_unit when using the constructor Dataset(your_path, voxel_size=your_voxel_size)."
```

Comment:
Further above it reads as though we want to allow both voxel_size and voxel_size_with_unit for the moment. Then I guess this warning should also happen only if both are missing (meaning the old keyword scale was used)?

Reply:
That is true, but I think it should already be the case. In line 261-268 the old arguments `scale` and `voxel_size` are converted to `voxel_size_with_unit`. When the `voxel_size_with_unit` variable is still not set, `scale` and `voxel_size` are unset too.
I am not sure about the purpose of the warning, but I suppose that it was possible to open an existing dataset with the Dataset constructor without specifying a voxel size in an old version of the libs.

Reply:
I see, thanks! Let’s keep it like this for now then. Maybe we can do a round of breaking changes (of stuff that has been deprecated for a while) some time soon.